Dockerfile.arm64

```diff
@@ -0,0 +1,20 @@
+
+FROM alpine:3.6 as alpine
+RUN apk add -U --no-cache ca-certificates mailcap
```

Comment:
And this mailcap package provides the mime.types file, so drop that as well

Reply:
yep removed both :)  thanks